packages/bindgen/src/realm_js_helpers.h

```diff
@@ -249,6 +250,10 @@ struct Helpers {
 
         return query.get_description() + ' ' + results.get_descriptor_ordering().get_description(query.get_table());
     }
+
+    static void feed_buffer(app::WatchStream& ws, const BinaryData& buffer) {
+        ws.feed_buffer({ buffer.data(), buffer.size() });
+    }
```

Comment:
Temporary helper until Core can provide an overload for `feed_buffer` (it currently only takes a `std::string_view`).

Reply:
Is there a core issue for it? If so, a link would remind us to check from time to time. If not, please create one.

Reply:
@RedBeard0531 will create one once he goes through all our helpers here to see if anything else could be simplified from Core's side as well.